src/pages/partners/payment-agent/_who-can-apply.tsx

```diff
@@ -70,6 +75,15 @@ const ButtonWrapper = styled(CenteredSection)`
     display: flex;
     justify-content: center;
 `
+const StyledListHeader = styled(Header)`
+    margin-top: 4rem;
+`
+const WhoCanApplyHeader = styled(Header)`
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 24px;
+`
```

Comment:
You shouldn't need this with the `type` prop. Please also include the `as` prop to your Headers otherwise the default is `h1` and multiple `h1`s on the page is bad for SEO. Use `weight` prop for font weights.

Reply:
Resolve this. Didn't understand why just 'w='normal'' doesn't work :D